src/index.js

```diff
@@ -42,7 +42,7 @@ const data = {
   cachingEnabled: false,
   throttle: {enabled: false, rate: 0}, // rate is in kBps
   proxyStatus: 'working',
```

Comment:
We could make 'online' part of the `proxyStatus` "enum". Then "online" is always used.
`proxyMessage` is if something is wrong, and there should be more description other than "failed" or "partial"